### ISSUE TITLE
In Internet Explorer 11, NVDA can't activate links on a particular website

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4796]
**Reported by k_kolev1985 on 2015-01-13 16:55**
It was reported to me and I have confirmed it, that with Internet Explorer 11, NVDA can't activate links on the following bulgarian website:
http://www.varna.utre.bg
When the page is opened in Firefox (v34.0.5) or Google Chrome, there is no such issue - NVDA activates links just fine. This occurs with both the latest "next" snapshot of NVDA and with the v2014.4 stable release. When the page is opened in Internet Explorer 11 however, pressing Enter or spacebar or trying to activate links via NVDA+Enter with the object navigation, nothing happens - NVDA doesn't even play its error sound. Actually, NVDA's object navigation on this website acts kind of weird anyway. I guess it is a problem of the website itself, but why only in Internet Explorer and not in Firefox or Chrome?

Steps to reproduce it:
1. Launch Internet Explorer 11.
2. Load the following website in it: http://www.varna.utre.bg.
3. Try to activate a link on the page with Enter, spacebar or NVDA+Enter.

Actual results:
NVDA fails to activate any link on that website with Internet Explorer, but it activates them just fine in Firefox and Chrome.

Expected results:
NVDA should open those links in Internet Explorer as it does in Firefox and Chrome.

Note: JAWS doesn't have this issue even in Internet Explorer with that website, but I don't know why. I haven't tested with Narrator.

**Comment by ehollig** [comment]
I am able to activate the links on this page using IE 11 and performing the steps described in https://github.com/nvaccess/nvda/issues/4796#issue-116047358. CC @leonardder

**Comment by ehollig** [comment]
@k_kolev1985, is this still an issue for you? Unlike my report in https://github.com/nvaccess/nvda/issues/4796#issuecomment-318408337, I am able to activate links on this page in IE11 using enter. If there are links that do not work for you, could you specify which ones?

**Comment by k-kolev1985** [comment]
I've done a quick test and now I can activate links on that website. Apparently there was a change either in NVDA or the website that made it possible. So, I think we can close this ticket.

**Comment by ehollig** [comment]
Thank you for the reply. Closing as worksforme
